lizmap/modules/lizmap/classes/lizmapLogConfig.class.php

```diff
@@ -9,40 +9,24 @@
  *
  * @license Mozilla Public License : http://www.mozilla.org/MPL/
  */
-class lizmapLogConfig
-{
-    // Lizmap log configuration data
-    private $data = array();
 
-    // general properties
-    private $properties = array(
-        'active',
-        'profile',
-    );
+use Lizmap\Logger as Log;
 
-    // list of the logItems of the ini file
-    protected $logItems = array();
-    // If the log is active globally or not
-    private $active = '';
-
-    // database profile
-    private $profile = '';
+class lizmapLogConfig
```

Comment:
mark it as deprecated. Probably we could remove it completely. We should inspect external modules to see if this class and lizmapLogItem are used directly.